arkade5/start.sh

```diff
@@ -36,8 +36,8 @@ for DIRECTORY in /opt/output/Arkade-testrapporter_*; do
         mv -v /opt/output/arkade_output.html "$ERROR_PATH"
         echo "Arkade did not create a report. Log is available at $ERROR_PATH"
     else
-        mv -v "$DIRECTORY/" "$OUTPUT_PATH_LOG"
-        echo "Arkade report is available at $OUTPUT_PATH_LOG"
+        mv -v "${DIRECTORY}"/* "$OUTPUT_PATH_LOG"
```

Comment:
```suggestion
        mv -v "${DIRECTORY}/*" "$OUTPUT_PATH_LOG"
```
Jeg trur dette burde funke greit også, men ta å test det først.

Reply:
Det funker ikke, har testet :)
`/*` blir ikke evaluerat så man sender at filnavnet er "/opt/...{UUID}/*"

Reply:
I see 😄